Bottom: 109386ac25479151eb665e0706b899c45ffd0f76
Top:    c9d7595b50e046eff2d055aa5c34f93b7292fd2d
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 12:38:12 +0000

Refresh of continue-tinkering-with-3d

---

diff --git a/src/hid/common/face3d.h b/src/hid/common/face3d.h
index a4d81a4..c630c17 100644
--- a/src/hid/common/face3d.h
+++ b/src/hid/common/face3d.h
@@ -1,5 +1,6 @@
 typedef struct {
   double nx, ny, nz; /* Face normal?*/
+  bool surface_orientation_reversed;
   GList *contours;
 
   /* For cylindrical surfaces */
@@ -12,7 +13,6 @@ typedef struct {
 
   /* XXX: STEP specific - breaks encapsulation */
   int surface_identifier;
-  bool surface_orientation_reversed;
   int face_identifier;
   int face_bound_identifier;
 } face3d;
